### DISCUSSION TITLE
ClickHouse sink

### DISCUSSION THREAD

**Discussion by ondar** [cybertec-postgresql/pgwatch#552] [Q&A] [answered]
Anyone know if I can use postgres sink (or any other available) for [https://github.com/ClickHouse](https://github.com/ClickHouse)? Or what are the diffrerences?

**Comment by pashagolub** [comment, marked as answer]
No, you cannot use postgres sink because Clickhouse doesn't speak Postgresian. 
But you can:
- export to json and then [import](https://clickhouse.com/docs/en/faq/integration/json-import) to clickhouse
- use a [special CDC](https://clickhouse.com/docs/en/integrations/postgresql) to read data from postgres
- write your own RPC sink implementation or use [this one](https://github.com/destrex271/pgwatch3_rpc_server/tree/main/cmd/clickhouse_receiver) as a starting point
